### PR TITLE
Dashboard

### PR DESCRIPTION
**Pull request by Joamena**
Agregar un container Fluid para agregar responsive al dashboard, actulizacion de dependencias para trabajar en datapicker

### CODE REVIEW

**Comment by tebanhdez** [comment]
@Emmurillo could you please take a look at this PR when you have the chance? It's related to UI so I think you're more suitable reviewer than me. 
Thanks.

File: src/main/bower.json [hunk 1]
```diff
@@ -10,11 +10,16 @@
     "angular-route": "^1.3.0",
     "angular-sanitize": "^1.3.0",
     "angular-touch": "^1.3.0",
-    "angular-ui-router": "ui-router#^0.2.18"
+    "angular-ui-router": "ui-router#^0.2.18",
+    "angular-bootstrap": "^1.3.2",
+    "ngDatepicker": "*"
```
Comment (emmurillo) [inline src/main/bower.json:15]: @Joamena it's kinda dangerous leave open the version of the package. Also noticed that you're requesting this dependency on the package.json as well, you should choose one and go with it.

File: src/main/webapp/app/dashboard/view.html [hunk 2]
```diff
@@ -1,4 +1,10 @@
-<header class="navbar shadow"></header>
-<sidebar></sidebar>
-<!-- <div ui-sref="dashboard.database"></div> -->
-<div class="dashboard-container" ui-view></div>
\ No newline at end of file
+<header class="navbar navbar-default navbar-fixed-top"></header>
+<div id="wrapper">
+	<div id="sidebar-wrapper" class="col-md-1">
```
Comment (emmurillo) [inline src/main/webapp/app/dashboard/view.html:3]: Too much spaces per tab.

File: src/main/webapp/app/dashboard/views/campaign/view.html [hunk 3]
```diff
@@ -1,5 +1,5 @@
  <div id="panelPosition" class="row">
-    <div class="col-md-4 col-md-offset-3">
+    <div class="col-md-4 col-md-offset-2 col-sm-4 col-xs-4">
```
Comment (emmurillo) [inline src/main/webapp/app/dashboard/views/campaign/view.html:2]: This whole line is equivalent to: `<div class="col-xs-4 col-md-offset-2">`

File: src/main/webapp/app/dashboard/views/campaign/view.html [hunk 4]
```diff
@@ -35,7 +35,7 @@
                 <form class="form-inline">
                     <div class="form-group">
                         <div class="row">
-                            <div class="col-md-4 col-md-offset-8">
+                            <div class="col-md-4 col-md-offset-4 col-xs-offset-2 col-xs-4 col-lg-3">
```
Comment (emmurillo) [inline src/main/webapp/app/dashboard/views/campaign/view.html:38]: Same here

File: src/main/webapp/app/dashboard/views/metrics/metrics.controller.js [hunk 5]
```diff
@@ -0,0 +1,13 @@
+(function() {
+  'use strict';
+
+  angular
+    .module('refiereApp.dashboard')
+    .controller('MetricsCtrl', MetricsCtrl);
+
+    MetricsCtrl.$inject = ['jkuri.datepicker'];
```
Comment (emmurillo) [inline src/main/webapp/app/dashboard/views/metrics/metrics.controller.js:8]: You can't inject module names to a controller, how does this even works?

File: src/main/webapp/app/dashboard/views/metrics/metrics.controller.js [hunk 6]
```diff
@@ -0,0 +1,13 @@
...
+
+  function MetricsCtrl(ngDatepicker) {
```
Comment (emmurillo) [inline src/main/webapp/app/dashboard/views/metrics/metrics.controller.js:10]: This is doing nothing

File: src/main/webapp/app/dashboard/views/metrics/view.html [hunk 7]
```diff
@@ -1,124 +1,126 @@
...
+        <div class="visual">
+          <i class="fa fa-users">
+          </i>
+        </div>
+        <div class="details">
+          <div class="number">1349</div>
+          <div class="desc">Cantidad de Referidores</div>
+        </div>
+        <a class="more" href=#>
+          "View More"
```
Comment (emmurillo) [inline src/main/webapp/app/dashboard/views/metrics/view.html:16]: Why are you adding double quotes?

File: src/main/webapp/app/dashboard/views/metrics/view.html [hunk 8]
```diff
@@ -1,124 +1,126 @@
...
+      </div>
+    </div>
+    <div id="PanelMetricsContainer" class="col-md-4">
+      <div class="dashboard-stat red">
+        <div class="visual">
+          <i class="fa fa-arrow-up">
+          </i>
+        </div>
+        <div class="details">
+          <div  class="number">12.5</div>
```
Comment (emmurillo) [inline src/main/webapp/app/dashboard/views/metrics/view.html:27]: @Joamena do you have a number class? this seems to be unnecessary.

File: src/main/webapp/app/dashboard/views/metrics/view.html [hunk 9]
```diff
@@ -1,124 +1,126 @@
...
+          <div class="desc">Referencias Efectivas</div>
```
Comment (emmurillo) [inline src/main/webapp/app/dashboard/views/metrics/view.html:28]: Ditto

**Comment by emmurillo** [comment]
@tebanhdez fix this PR would take more time than start it from scratch, will close it, @Joamena should open a new one.  As an additional comment, this one is addressing way too many tasks in one single PR which makes the review process painful.

**Pull request closed without merging**